infra/front-door/azure-waf.bicep

```diff
@@ -0,0 +1,91 @@
+@description('Required. The name of the Front Door endpoint to create. This must be globally unique.')
```

Comment:
is this description correct? It is saying the name of "Front Door endpoint".

Reply:
Updated